core/templates/dev/head/domain/utilities/AutogeneratedAudioLanguageObjectFactorySpec.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for AutogeneratedAudioLanguageObjectFactory.
+ */
+
+describe('AutogeneratedAudioLanguage object factory', function() {
+  var autogenAudioLanguage = null;
+  var aalof = null;
+  beforeEach(module('oppia'));
+  beforeEach(inject(function($injector) {
+    aalof = $injector.get('AutogeneratedAudioLanguageObjectFactory');
+
+    autogenAudioLanguage = aalof.createFromDict({
+      id: 'a',
+      text: 'a description',
```

Comment:
Interesting discrepancy between "text" and "description". I did a little digging and it seems like the name changes between AUTOGENERATED_AUDIO_LANGUAGES in constants.js and here.

OK to merge this PR, but would it be possible to do a follow-up that standardizes it to "description" everywhere? I.e. update createFromDict() and also go back to where the dicts came from and change the key there to 'description' instead of 'text'?

Thanks!

Reply:
Will address changing attribute name in a different PR for [AudioLanguage.js](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/domain/utilities/AudioLanguageObjectFactory.js) and [AutogenAudioLanguage.js](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/domain/utilities/AutogeneratedAudioLanguageObjectFactory.js) , possibly in [constants.js](https://github.com/oppia/oppia/blob/develop/assets/constants.js) too

Reply:
Sounds great, thanks @hoangviet1993!